Using `bicep` as part of path name throws an error

<!-- Please read our Rules of Conduct: https://opensource.microsoft.com/codeofconduct/ -->
<!-- Please search existing issues to avoid creating duplicates. -->

**Describe the bug**
The replace logic to find the transpiled bicep template can't handle paths where `bicep` is part of the path. It will replace `bicep` in all places in the path.

**Steps to reproduce**

1. Create a resource group or bicep template file with `bicep` in the file name
2. Run Push
3. The push will fail

```
[12:41:43][Invoke-AzOpsPush] Deployment required
[12:41:43][Invoke-AzOpsPush] Adding or modifying:
[12:41:43][Invoke-AzOpsPush]   root/ivemo (ivemo)/platform (platform)/management (management)/ivemo-platform-management (***)/bicep-modules-rg/mybicepmodulesacr.bicep
[12:41:43][Invoke-AzOpsPush] Deleting:
Could not find a part of the path '/home/vsts/work/1/s/root/ivemo (ivemo)/platform (platform)/management (management)/ivemo-platform-management (***).json-modules-rg/m.jsonmodulesacr.json'.WARNING: [12:41:48][New-AzOpsScope] Path not found: root/ivemo (ivemo)/platform (platform)/management (management)/ivemo-platform-management (***).json-modules-rg/m.jsonmodulesacr.json
[12:41:48][Invoke-AzOpsPush] Expecting deploymentList object, it was not found. Exiting runtime with error code 1.
##[error]PowerShell exited with code '1'.
Finishing: Validate
```

**Screenshots**

 
![image](https://user-images.githubusercontent.com/17688234/165091473-3d8faa88-ab22-40fd-baff-b8997c75ca09.png)

--- a/src/functions/Invoke-AzOpsPush.ps1
+++ b/src/functions/Invoke-AzOpsPush.ps1
@@ -236,7 +236,7 @@
             }
 
             # Handle Bicep templates
-            if ($addition.EndsWith(".bicep")) { 
+            if ($addition.EndsWith(".bicep")) {
                 Assert-AzOpsBicepDependency -Cmdlet $PSCmdlet
                 $transpiledTemplatePath = $addition -replace '\.bicep', '.json'
                 Write-PSFMessage -Level Verbose @common -String 'Invoke-AzOpsPush.Resolve.ConvertBicepTemplate' -StringValues $addModifySet, $transpiledTemplatePath

--- a/src/functions/Invoke-AzOpsPush.ps1
+++ b/src/functions/Invoke-AzOpsPush.ps1
@@ -236,9 +236,9 @@
             }
 
             # Handle Bicep templates
-            if ($addition.EndsWith(".bicep")) {
+            if ($addition.EndsWith(".bicep")) { 
                 Assert-AzOpsBicepDependency -Cmdlet $PSCmdlet
-                $transpiledTemplatePath = $addition -replace '.bicep', '.json'
+                $transpiledTemplatePath = $addition -replace '\.bicep', '.json'
                 Write-PSFMessage -Level Verbose @common -String 'Invoke-AzOpsPush.Resolve.ConvertBicepTemplate' -StringValues $addModifySet, $transpiledTemplatePath
                 Invoke-AzOpsNativeCommand -ScriptBlock { bicep build $addition --outfile $transpiledTemplatePath }
                 $addition = $transpiledTemplatePath

